### PR TITLE
XW-4144 | SEOTweaks tweaks

### PR DESCRIPTION
**Pull request by hakubo**
…weaks

### CODE REVIEW

File: extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php [hunk 1]
```diff
@@ -16,157 +16,180 @@ class SEOTweaksHooksHelper {
...
+			$name = $first[0]['name'];
+
+			return Title::newFromText( $name, NS_FILE );
+		}
+
+		return null;
+	}
+
+	public static function onArticleSaveComplete( &$article ) {
+		if ( !empty( $article ) && $article instanceof Article ) {
```
Comment (macbre) [inline extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php]: `instanceof` check is enough

File: extensions/wikia/ImageServing/imageServing.class.php [hunk 2]
```diff
@@ -99,97 +99,100 @@ public function getRequestedHeight() {
...
 			// apply limiting
-			foreach($out as &$entry) {
-				$entry = array_slice($entry, 0, $limit);
+			foreach ( $out as &$entry ) {
+				$entry = array_slice( $entry, 0, $limit );
 			}
 		}
 
-		wfProfileOut(__METHOD__);
+		var_dump($out);
```
Comment (macbre) [inline extensions/wikia/ImageServing/imageServing.class.php]: Please remove

File: extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php [hunk 3]
```diff
@@ -143,7 +142,8 @@ function () use ( $namespace, $title ) {
 					// Even if there is no og:image, we store the info in memcahe so we don't do the
 					// processing again
 					return '';
-				}
+				},
+				WikiaDataAccess::REFRESH_CACHE
```
Comment (macbre) [inline extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php]: Why do we want to ignore cache here?

```
WikiaDataAccess::REFRESH_CACHE - is equivalent of mcache=writeonly for one variable - use this option wisely
```

**Review by macbre — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/ImageServing/imageServing.class.php [hunk 4]
```diff
@@ -190,8 +190,6 @@ public function getImages( $limit = 5, $driverName = null ) {
 			}
 		}
 
-		var_dump($out);
```
Comment (hakubo) [inline extensions/wikia/ImageServing/imageServing.class.php]: no nono

**Pull request merged**